theano/configdefaults.py

```diff
@@ -1,11 +1,19 @@
+import errno
 import os
 import sys
 import logging
+import numpy
+import platform
+import textwrap
+import re
+import socket
 
 import theano
 from theano.configparser import (AddConfigVar, BoolParam, ConfigParam, EnumStr,
                                  FloatParam, IntParam, StrParam,
-                                 TheanoConfigParser)
+                                 TheanoConfigParser, THEANO_FLAGS_DICT)
+from theano.gof.compiledir import (local_bitwidth, python_int_bitwidth,
+                                   gcc_version_str)
```

Comment:
You should probably move those in rather than import them.

Since they are used elsewhere, you can either import them in their original location from here, or change the imports.